linera-summary/src/github.rs

```diff
@@ -164,20 +166,66 @@ impl Github {
         &self.context
     }
 
-    pub async fn comment_on_pr(&self, body: String) -> Result<()> {
+    pub async fn upsert_pr_comment(&self, body: String) -> Result<()> {
```

Comment:
Optional nit: I haven't seen "upsert" used before (but that may be on me), so I'd suggest adding some documentation as well :sweat_smile: 

```suggestion
    /// Updates an existing comment or creates a new one on the PR.
    pub async fn upsert_pr_comment(&self, body: String) -> Result<()> {
```

Reply:
I think it's used more in the context of DBs 😅 but it was the only way I could think of naming this that wasn't too long ha
I'll add the comment!